definitions/v7/lat-team-api.yml

```diff
@@ -181,9 +163,7 @@ search:
       case:
         0: 1 # normal
         1: 2 # double
-    minimumratio:
```

Comment:
`# minimumratio: ratioless`

Reply:
I'm not sure I understand this. The site has a global minimum ratio of 0.4, but torrents must be seeded for 2 days regardless of ratio. Does this field refer to individual torrents or the global ratio?

Reply:
similar sites use language like this; edited to fit latt so you can copy paste

```
# global MR is 0.4 but torrents must be seeded for 2 days regardless of ratio
#    minimumratio:
#      text: 0.4
```

Reply:
Thanks, I've added it to the file